<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>all_file_dialog</class>
 <widget class="QDialog" name="all_file_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import all result files</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>22</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="bxsf_layout">
      <item>
       <widget class="QPushButton" name="bxsf_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>BXSF file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="bxsf_label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="long_layout">
      <item>
       <widget class="QPushButton" name="long_button">
        <property name="text">
         <string>results_long.out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="long_label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="ang_layout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="ang_button">
        <property name="text">
         <string>results_orbitoutlines_invAng.out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ang_label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="au_layout">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="au_button">
        <property name="text">
         <string>results_orbitoutlines_invau.out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="au_label">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="import_button">
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
